lang/en.js

```diff
@@ -67,6 +67,15 @@ var language = {
       'title':'Send us your feedback about the Python Editor',
       'label':'Send Feedback'
     },
+    'en': {
+      'title': 'English'
+    },
+    'es': {
+      'title': 'Spanish'
+    },
+    'pl': {
+      'title': 'Polish'
+    },
```

Comment:
Since this list will grow over time it makes sense to group them in one key. Could we have something like

```
languages: {
  'en': 'English',
  'es': 'Spanish',
  ...
}
```